scripts/main.js

```diff
@@ -35,4 +40,85 @@ document.addEventListener('DOMContentLoaded', () => {
+    links.innerHTML = `
+<a href="${project.liveVersion || '#'}" target="blank">
+  <span>See Live </span
+  ><img src="./images/icons/go-live.svg" alt="see live" />
+</a>
+<a href="${project.source || '#'}" target="blank">
+  <span>See Source </span>
+  <img src="./images/icons/GitHub-white.svg" alt="see source" />
+</a>`;
+    project.technologies.forEach((list) => {
+      const newLi = list.cloneNode(true);
+      technoList.appendChild(newLi);
+    });
+    title.setAttribute('id', 'title');
+    title.textContent = `${project.title} --- ${project.name}`;
+    featureImage.setAttribute('src', project.featuredImage);
+    featureImage.setAttribute('alt', project.featuredImageAlt);
+    technology.setAttribute('id', 'technology');
+    technology.appendChild(technoList);
+    description.classList.add('description');
+    links.classList.add('links');
+    description.textContent = project.description;
+    recentWorkModal.querySelector('#feature').appendChild(featureImage);
+    recentWorkModal.querySelector('.container').appendChild(title);
+    recentWorkModal.querySelector('.container').appendChild(technology);
+    recentWorkModal.querySelector('.container').appendChild(description);
+    recentWorkModal.querySelector('.container').appendChild(links);
+    recentWorkModal.classList.remove('display-none');
+    fullPage.setAttribute('style', `height: ${recentWorkModal.scrollHeight}px`);
+    dispose.addEventListener('click', () => hideModal());
+    disposeDesktop.addEventListener('click', () => hideModal());
+  };
+
+  // create Array having list of projects
+  const recentWork = document.querySelector('#recent-work');
+  const ProjectObjects = recentWork.querySelectorAll('.card');
+  const Projects = [];
+  const featureImageType = window.innerWidth < 922 ? '-mobile.svg' : '.svg';
+  ProjectObjects.forEach((project) => {
+    const projectObject = {
+      name: project.getAttribute('data-name'),
+      title: project.querySelector('.project-title').textContent,
+      description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. In varius
+      turpis id metus vehicula, a faucibus neque vehicula. Etiam
+      tincidunt ante et dui efficitur ultricies. Nulla ex felis, mattis
+      eget lacinia sed, molestie at nisi. Nulla iaculis mi finibus augue
+      pharetra, quis pellentesque metus hendrerit. Vestibulum tristique
+      sapien eu velit porttitor semper. Aliquam sed elementum enim.
+      Suspendisse ultrices quis enim at gravida. Ut lectus urna, cursus
+      et tellus in, faucibus lacinia urna. Morbi nibh purus, vehicula at`,
+      featuredImage: project.getAttribute('data-image') + featureImageType,
+      featuredImageAlt: `${project.getAttribute('data-name')} Image`,
+      technologies: Array.from(project.querySelectorAll('li')),
+      liveVersion: project.getAttribute('data-live'),
+      source: project.getAttribute('data-source'),
+    };
```

Comment:
- [ ] You did well using this approach. However, you should have created an array of objects, and later on, iterate through the array and render all the object properties in the popup modal.